docs/source/upcoming_release_notes/614-LAMP_motion_system_class.rst

```diff
@@ -0,0 +1,33 @@
+614 LAMP motion system class
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- LAMP motion Class for the LAMP endstation TMO. This indluces the following motion axes:
```

Comment:
```suggestion
- LAMP motion Class for the LAMP endstation TMO. This includes the following motion axes:
```

Reply:
Thanks for catching that! Resolved in commit 5d3e726